openmls/tests/test_external_commit.rs

```diff
@@ -120,3 +120,157 @@ fn test_external_commit(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoPr
         assert_eq!(got_error, ExternalCommitError::InvalidGroupInfoSignature);
     }
 }
+
+#[apply(ciphersuites_and_backends)]
+fn test_group_info(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider) {
+    // Alice creates a new group ...
+    let mut alice_group = {
+        let group_config = MlsGroupConfigBuilder::new()
+            .use_ratchet_tree_extension(true)
+            .crypto_config(CryptoConfig::with_default_version(ciphersuite))
+            .build();
+
+        let alice_cb = {
+            let alice_cb = CredentialBundle::new(
+                b"Alice".to_vec(),
+                CredentialType::Basic,
+                ciphersuite.signature_algorithm(),
+                backend,
+            )
+            .expect("Creation of credential bundle failed.");
+
+            let index = alice_cb
+                .credential()
+                .signature_key()
+                .tls_serialize_detached()
+                .expect("Serialization of signature public key failed.");
+
+            backend
+                .key_store()
+                .store(&index, &alice_cb)
+                .expect("Storing of signature public key failed.");
+
+            alice_cb
+        };
+
+        MlsGroup::new(
+            backend,
+            &group_config,
+            alice_cb.credential().signature_key(),
+        )
+        .expect("An unexpected error occurred.")
+    };
+
+    // Self update Alice's to get a group info from a commit
+    let (.., group_info) = alice_group.self_update(backend, None).unwrap();
+    alice_group.merge_pending_commit().unwrap();
```

Comment:
Maybe it would be worth to also return the GroupInfo when creating a new group? We don't have to do it in this PR, though.